tests/images/clusterloader2/Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM golang:1.16.4 AS builder
 WORKDIR /go/src/k8s.io
 RUN git clone https://github.com/kubernetes/perf-tests
+WORKDIR perf-tests
+RUN git fetch origin --verbose --tags
+RUN git checkout $branch
```

Comment:
How does the $branch variable get set? Do you set it on your terminal before running docker build? Don't we need an `ARG`?
https://stackoverflow.com/a/51770027/3175021

Reply:
`docker build --build-arg branch=release-1.23 .` - this is how it is run. It's a docker feature to be able to pass build args.

Reply:
Can we document this somewhere in the repo in the docs?